### ISSUE TITLE
Undefined method start for nil class

### ISSUE THREAD

**Issue by fearnowrath** [mojombo/chronic#338]
When i try to use `Chronic.parse` on the following I get an error.
```
require 'chronic'
puts Chronic.parse '9 hours and 43 minutes ago'
```
Error:
`/chronic/handlers.rb:517:in `get_anchor': undefined method `start=' for nil:NilClass (NoMethodError)`

What could be causing the error?

**Comment by SagareGanesh** [comment]
@fearnowrath I think we don't have option like _**xx hours and xx minutes ago**_ . 
Instead of this you can use _**xx hours before**_ or _**xx minutes before**_.
In your case you can use _**583 minutes before**_. (9 hours and 43 minutes = 583 minutes)

**Comment by fearnowrath** [comment]
Thanks but why does it throw an error rather than just returning nil? Is it possible to at least make that return nil? Also any plans on adding support for this kind of format in the future?

**Comment by davispuh** [comment]
yeah, that is also a bug, and it should be pretty easy to support for this once #278 is finished

**Comment by arezk84** [comment]
I have the same error
`NoMethodError: undefined method `start=' for nil:NilClass`
If I just try to parse the character "t"
`Chronic.parse("t")`

Is there any updates?

**Comment by centerpush** [comment]
I'm looking into solving this problem at the moment. Before I dig too deep into the code, I just wanted to see if there was any interest in resolving it or if we're fine with the workaround provided by @SagareGanesh.
